pkg/kopia/command/server.go

```diff
@@ -60,6 +62,8 @@ func ServerStart(cmdArgs ServerStartCommandArgs) []string {
 		args = args.AppendLoggable(readOnlyFlag)
 	}
 
+	args = cmdArgs.kopiaCacheArgs(args, cmdArgs.CacheDirectory)
+
```

Comment:
[nit] Let's move this after `args := commonArgs()` on line no 41.

Reply:
Done !